core/miner/impl/miner_impl.cpp

```diff
@@ -56,7 +62,8 @@ namespace fc::miner {
       std::shared_ptr<Manager> sector_manager,
       std::shared_ptr<Scheduler> scheduler,
       std::shared_ptr<boost::asio::io_context> context,
-      mining::Config config) {
+      mining::Config config,
+      std::vector<Address> precommit_control) {
```

Comment:
I don't see where it is used

Reply:
it shouldn't be used for now, it will be needed after the commit batching will be added

Reply:
In this case
```suggestion
      const std::vector<Address> &precommit_control) {
```

And add TODO comment

Reply:
Or add to the commit batching PR.

Reply:
Please, pay attention to the comment.